src/config.rs

```diff
@@ -135,17 +174,21 @@ impl<T: near_workspaces::Network> From<near_workspaces::Worker<T>> for NetworkCo
             linkdrop_account_id: None,
             near_social_db_contract_account_id: None,
             faucet_url: None,
-            meta_transaction_relayer_url: None,
             fastnear_url: None,
+            meta_transaction_relayer_url: None,
             staking_pools_factory_account_id: None,
         }
     }
 }
 
 #[derive(Debug)]
+/// Represents the possible outcomes of a retryable operation.
 pub enum RetryResponse<R, E> {
```

Comment:
this type doesn't really bubble up to be eventually visible in documentation, does it

Reply:
For now, I just not really sure if it's needed to be public. But I kept documentation just in case I will decide to make it public